test/01_type/sec_to_time_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test/unit'
+require_relative '../../src/01_type/sec_to_time'
+
+class TypeTest < Test::Unit::TestCase
+  def test_sec_to_time_cond1
+    actual = round_ones_place(46_979)
```

Comment:
Oh! I forgot to change the function call after copying template.
OK now I understand all his points must be fixed, thanks :)